compiler/noirc_evaluator/src/ssa/ir/value.rs

```diff
@@ -27,16 +27,27 @@ pub(crate) enum Value {
     /// Example, if you add two numbers together, then the resulting
     /// value would have position `0`, the typ would be the type
     /// of the operands, and the instruction would map to an add instruction.
-    Instruction { instruction: InstructionId, position: usize, typ: Type },
+    Instruction {
+        instruction: InstructionId,
+        position: usize,
+        typ: Type,
+    },
 
     /// This Value originates from a block parameter. Since function parameters
     /// are also represented as block parameters, this includes function parameters as well.
     ///
     /// position -- the index of this Value in the block parameters list
-    Param { block: BasicBlockId, position: usize, typ: Type },
+    Param {
+        block: BasicBlockId,
+        position: usize,
+        typ: Type,
+    },
```

Comment:
Odd that the formatter touched this when it hasn't changed? Only a new variant was added. Maybe it is the lack of a doc comment on it? Odd.

Reply:
Either way I'll add a doc comment to the global variant.

Reply:
Looks like the formatter acted as it did previously with a new doc comment.